src/topology/continuous_function/algebra.lean

```diff
@@ -68,7 +68,7 @@ In this section we show that continuous functions valued in a topological group
 the structure of a group.
 -/
 
-section subtype
+section subtype -- deprecated
```

Comment:
The docstring at the top of this file does not mention that this is deprecated, so I'd be inclined not to add these comments.

Reply:
I think that @semorrison agreed to add that the subtype was deprecated but at the time I did not want to add it and I do not remember why. I changed my mind now, so if @semorrison agrees I will also add a comment in the file header.

Reply:
If you go ahead with this, then you should update the PR title and description to indicate that you're deprecating them.

Frankly though I'd just make these as entirely separate PRs, it make both reviewing and the resulting git history cleaner.